EasyNetQ.DI.Microsoft cannot be used in Asp.Net Core 2.1 project

Hi,

I am trying to use `EasyNetQ` in Asp.Net Core 2.1 project together with `EasyNetQ.DI.Microsoft`.

Unfortunately, I cannot add `EasyNetQ.DI.Microsoft` to Asp.Net Core 2.1 project since it depends on `Microsoft.Extensions.DependencyInjection v2.2.0` which conflicts with `Microsoft.AspNetCore.App v2.1.*`.

Here's an error I'm getting:
```
NU1107: Version conflict detected for Microsoft.Extensions.DependencyInjection. Install/reference Microsoft.Extensions.DependencyInjection 2.2.0 directly to project EasyNetQ.AspNetCore2_1 to resolve this issue. 
 EasyNetQ.AspNetCore2_1 -> EasyNetQ.DI.Microsoft 6.0.1 -> Microsoft.Extensions.DependencyInjection (>= 2.2.0) 
 EasyNetQ.AspNetCore2_1 -> Microsoft.AspNetCore.App 2.1.1 -> Microsoft.Extensions.DependencyInjection (>= 2.1.1 && < 2.2.0).
```

I did a bit of digging and seems like this behavior was changed in the version 3.7.0 where `Microsoft.Extensions.DependencyInjection` was upgraded from 2.1.0 to 2.2.0.

Both EasyNetQ and EasyNetQ.DI.Microsoft target netstandard2.0, so I wonder if the drop of Asp.Net Core 2.1 was intentional.

There is a workaround of re-implementing the code of `EasyNetQ.DI.Microsoft` in projects that target Asp.Net Core 2.1 but having support in the official package would be nice.

P.S. Thanks for maintaining this awesome library.

`Microsoft.AspNetCore.App 2.2+` is not compatible with `netcoreapp2.1`.
This requires targeting `netcoreapp2.2` or higher which means that `EasyNetQ.DI.Microsoft` is not compatible with Asp.Net Core 2.1.

```
error NU1202: Package Microsoft.AspNetCore.App 2.2.0 is not compatible with netcoreapp2.1 (.NETCoreApp,Version=v2.1). Package Microsoft.AspNetCore.App 2.2.0 supports: netcoreapp2.2 (.NETCoreApp,Version=v2.2)
```

The reason for creating this issue was to ask if dropping asp.net 2.1 was as designed or non-intentional and if it's possible to bring  support for asp.net 2.1 back to `EasyNetQ.DI.Microsoft`.

> `Microsoft.AspNetCore.App 2.2+` is not compatible with `netcoreapp2.1`.
> This requires targeting `netcoreapp2.2` or higher which means that `EasyNetQ.DI.Microsoft` is not compatible with Asp.Net Core 2.1.
> 
> ```
> error NU1202: Package Microsoft.AspNetCore.App 2.2.0 is not compatible with netcoreapp2.1 (.NETCoreApp,Version=v2.1). Package Microsoft.AspNetCore.App 2.2.0 supports: netcoreapp2.2 (.NETCoreApp,Version=v2.2)
> ```
> 
> The reason for creating this issue was to ask if dropping asp.net 2.1 was as designed or non-intentional and if it's possible to bring support for asp.net 2.1 back to `EasyNetQ.DI.Microsoft`.

Sorry, but it seems there is no other way to fix this except an upgrade to `netcoreapp2.2`, because in case of a downgrade of `Microsoft.Extensions.DependencyInjection` to `<2.2.0`, the same mentioned thing will happen with `Microsoft.AspNetCore.App 2.2.0` users.

--- a/Source/EasyNetQ.DI.Microsoft/EasyNetQ.DI.Microsoft.csproj
+++ b/Source/EasyNetQ.DI.Microsoft/EasyNetQ.DI.Microsoft.csproj
@@ -15,7 +15,7 @@
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
     <PackageReference Include="GitVersionTask" Version="5.0.1">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>

